katalog/prometheus-operator/deploy.yml

```diff
@@ -55,4 +64,5 @@ spec:
     port: 8080
     targetPort: http
   selector:
-    k8s-app: prometheus-operator
+    app.kubernetes.io/component: controller
```

Comment:
Are we sure about renaming labels? I don't know how it could affect to the rest of the components

Reply:
We have to be careful with this because updating the `selector` field in `Service` resource is not supported so it is implied to you have to recreate it.

Reply:
is this a simple "restore previous labels" or have we defined a new naming convention for labels? it's not clear to me from all the different comments, sorry

Reply:
No problem, it's a simple "restore previous labels".